tools/go.mod

```diff
@@ -5,12 +5,21 @@ go 1.18
 require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/mikefarah/yq/v4 v4.27.5
+	github.com/spf13/cobra v1.5.0
 	github.com/spf13/pflag v1.0.5
+	github.com/vmware-archive/yaml-patch v0.0.11
+	golang.org/x/tools v0.1.12
+	gopkg.in/yaml.v3 v3.0.1
+	k8s.io/apiextensions-apiserver v0.24.4
 	k8s.io/apimachinery v0.24.4
 	k8s.io/code-generator v0.24.4
 	k8s.io/klog/v2 v2.70.1
+	sigs.k8s.io/controller-tools v0.9.2
+	sigs.k8s.io/yaml v1.3.0
 )
 
+replace sigs.k8s.io/controller-tools => github.com/openshift/controller-tools v0.9.3-0.20220912174723-cf3ef054f3dd // v0.9.2+openshift-0.2
```

Comment:
Can we combine this with another replacement? Move code generator replacement here, perhaps

Reply:
Fixed